sdk/communication/Azure.Communication.Calling.Server/src/Models/CallState.Serialization.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.Calling.Server
+{
+    internal static partial class CallStateExtensions
```

Comment:
We should be using extensible enum instead of manually converting strings to enums